src/test/java/learn/field_agent/data/AliasJdbcTemplateRepositoryTest.java

```diff
@@ -13,7 +13,7 @@
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
 class AliasJdbcTemplateRepositoryTest {
 
-    private final int NEXT_ID = 5;
+    private final int NEXT_ID = 4;
```

Comment:
Only three aliases are being inserted in your known good state stored procedure so your next ID will be `4`.